draft-ietf-quic-recovery.md

```diff
@@ -412,7 +412,7 @@ is available, or if the network changes, the initial RTT SHOULD be set to 100ms.
 When an acknowledgement is received, a new RTT is computed and the timer
 SHOULD be set for twice the newly computed smoothed RTT.
 
-When crypto packets are sent, the sender MUST set a timer for the crypto
+Every time a crypto packet is sent, the sender MUST set a timer for the crypto
```

Comment:
How about "A timer MUST be updated to expire after the crypto timeout period after a new crypto packet is sent"?

Reply:
> If no previous RTT is available,
-->
If no previous RTT, or a-priori estimate, is available,

Reply:
@mikkelfj is this suggestion about a different paragraph?

Reply:
yes, I should have been more clear. It is existing text in the preceding paragraph, or thereabout.